botocore/session.py

```diff
@@ -196,8 +196,9 @@ def _register_data_loader(self):
     def _register_endpoint_resolver(self):
         def create_default_resolver():
             loader = self.get_component('data_loader')
-            endpoints = loader.load_data('endpoints')
-            return EndpointResolver(endpoints)
+            endpoints, path = loader.load_data('endpoints', return_path=True)
+            builtin = loader.is_builtin_path(path)
```

Comment:
nit: should be `uses_builtin` or `is_builtin`

Reply:
Why? The function checks if the string in the argument is a builtin path or not: `myloader.is_builtin_path("/path/to/my/file")`. I think the current name is more descriptive and specific.

Reply:
I disagree, just naming `builtin` implies that it is a built in and presents ambiguity. Including a verb in the name like `uses` or `is` indicates that it's a boolean. You included it in the function name after all 😉

Reply:
Oh, you are talking about the variable name. I thought this is about the method name...

Reply:
Ah sorry yeah the variable name!